apps/console/src/features/userstores/components/edit/edit-connection-details-user-store.tsx

```diff
@@ -283,89 +296,115 @@ export const EditConnectionDetails: FunctionComponent<EditConnectionDetailsProps
                         <Grid.Column width={ 8 }>
                             {
                                 properties?.required?.map((property: TypeProperty, index: number) => {
-                                    const name = property.description.split("#")[ 0 ];
-                                    const isPassword = property.attributes
-                                        .find(attribute => attribute.name === "type").value === "password";
-                                    const toggle = property.attributes
-                                        .find(attribute => attribute.name === "type")?.value === "boolean";
+                                    const name: string = property.description.split("#")[ 0 ];
```

Comment:
Let's add a null check here. Check in other places as well

Reply:
No code change was done here, but only a fix for exceeding line lengths. Therefore, we can fix this in a separate effort I think.